src/protocol.h

```diff
@@ -354,34 +354,38 @@ static inline bool MayHaveUsefulAddressDB(ServiceFlags services)
     return (services & NODE_NETWORK) || (services & NODE_NETWORK_LIMITED);
 }
 
+
 /** A CService with information about it as peer */
 class CAddress : public CService
 {
     static constexpr uint32_t TIME_INIT{100000000};
 
+    /** The disk serialization for CAddress includes a version number.
+     *  Traditionally the number CLIENT_VERSION was used, but it has since
+     *  been disentangled from it. */
+    static constexpr uint32_t DISK_VERSION{210000};
+
 public:
     CAddress() : CService{} {};
     explicit CAddress(CService ipIn, ServiceFlags nServicesIn) : CService{ipIn}, nServices{nServicesIn} {};
 
-    SERIALIZE_METHODS(CAddress, obj)
+    enum class Format {
+        DISK,
+        NETWORK_NOTIME,
+        NETWORK_WITHTIME,
+    };
+
+    SERIALIZE_METHODS_PARAMS(CAddress, obj, Format, fmt)
     {
         SER_READ(obj, obj.nTime = TIME_INIT);
-        int nVersion = s.GetVersion();
```

Comment:
One include can now be removed, see https://github.com/bitcoin/bitcoin/pull/19503#issuecomment-668585730